sdk/graphservices/Azure.ResourceManager.GraphServices/tests/Tests/AccountCRUDTests.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.GraphServices.Tests.Helpers;
+using Azure.ResourceManager.GraphServices.Models;
+using NUnit.Framework;
+using Azure.Core.TestFramework;
+
+namespace Azure.ResourceManager.GraphServices.Tests.Tests
+{
+    public class AccountCRUDTests : GraphServicesManagementTestBase
+    {
+        public AccountCRUDTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [RecordedTest]
+        public async Task TestAccountCRUDOperations()
+        {
+            // Get Default Subscription
+            SubscriptionResource subscription = await this.Client.GetDefaultSubscriptionAsync();
+            ResourceGroupCollection resourceGroups = subscription.GetResourceGroups();
+
+            // Create Resource Group
+            string resourceGroupName = "myRgName";
+            ResourceGroupResource resourceGroup = await this.CreateResourceGroup(subscription, resourceGroupName, Core.AzureLocation.WestUS);
+            GraphServicesAccountResourceCollection collection = resourceGroup.GetGraphServicesAccountResources();
+
+            // Set Account Resource Values
+            var resourceName = Recording.GenerateAssetName("SDKAccounts");
+            string appId = new AccountTestEnvironment().ApplicationIDClient;
```

Comment:
we are having some issues in the playback of this test case.
I am not very familiar with this feature, but here we cannot just create a new test environment. We should already have an instance in the test base because it inherits the generic version `RecordedTestBase<TestEnvironment>`, something like:
```
var appId = TestEnvironment.ApplicationIDClient;
```